data/styles/vehicle/include/priorities_3_FG.prio.txt

```diff
@@ -274,6 +274,8 @@ highway-track-tunnel                                # line z16-
 === 130
 
 highway-construction                                # line z13-
+psurface-unpaved_bad                                # line z14-
+psurface-unpaved_good                               # line z14-
```

Comment:
Do we need it?
- These types are not in mapcss styles and not in out drules
- There are 4 psurface subtypes

Reply:
UPD: Aha, actually they are in the vehicle style. Could you please show some screenshots, I am just curious how it looks like :)

Reply:
I suspect that these line styles should be _higher_ than all highway=* lines.

Reply:
So, can we move it higher and see the picture?

Reply:
Yes, but I don't want to do it in this PR :)
Let's keep it simple and about validation only.